taxi/forms.py

```diff
@@ -0,0 +1,64 @@
+from django import forms
+from django.contrib.auth import get_user_model
+from django.contrib.auth.forms import UserCreationForm
+from django.core.exceptions import ValidationError
+
+from taxi.models import Driver, Car
+
+
+class DriverCreationForm(UserCreationForm):
+
+    class Meta(UserCreationForm.Meta):
+        model = Driver
+        fields = UserCreationForm.Meta.fields + ("license_number",)
+
+
+class DriverLicenseUpdateForm(forms.ModelForm):
+    MAX_LEN = 8
+
+    class Meta:
+        model = Driver
+        fields = ("license_number",)
+
+    def clean_license_number(self):
+        license_number = self.cleaned_data["license_number"]
+
+        if len(license_number) != DriverLicenseUpdateForm.MAX_LEN:
+            raise ValidationError(
+                f"Ensure that length of license number "
+                f"is < than {DriverLicenseUpdateForm.MAX_LEN}!"
+            )
+
+        license_number_triad = license_number[:3]
+        is_lower = license_number_triad.islower()
+        is_str = isinstance(license_number_triad, str)
+        is_num = False
+        for num_ in range(10):
+            if str(num_) in license_number_triad:
+                is_num = True
```

Comment:
<img width="222" alt="image" src="https://github.com/mate-academy/py-taxi-service-user-form-and-validation/assets/65166203/bfa95eb8-dbdc-4ab9-a14e-e6405bb6e885">